activesupport/lib/active_support/backtrace_cleaner.rb

```diff
@@ -3,33 +3,21 @@
 module ActiveSupport
   # = Backtrace Cleaner
   #
-  # Backtraces often include many lines that are not relevant for the context
-  # under review. This makes it hard to find the signal amongst the backtrace
-  # noise, and adds debugging time. With a BacktraceCleaner, filters and
-  # silencers are used to remove the noisy lines, so that only the most relevant
-  # lines remain.
+  # BacktraceCleaner holds filters and silencers which will inspect and mutate line-by-line the application's backtrace.
   #
-  # Filters are used to modify lines of data, while silencers are used to remove
-  # lines entirely. The typical filter use case is to remove lengthy path
-  # information from the start of each line, and view file paths relevant to the
-  # app directory instead of the file system root. The typical silencer use case
-  # is to exclude the output of a noisy library from the backtrace, so that you
-  # can focus on the rest.
+  # \Rails applications are configured by default to silence the backtraces of gems and standard libraries.
```

Comment:
I can see this being a replacement for the last paragraph since it talks about `remove_{silencers,filters}`, but I think removing the paragraph about how filters and silencers work is removing valuable information.